docs/modules/ic-identity-guide/pages/what-is-ic-identity.adoc

```diff
@@ -6,37 +6,30 @@
+_Internet Identity_ is a smart contract implemented on the {IC} blockchain. It enables you to authenticate securely and anonymously when you access dapps on the {IC}. To authenticate, one must specify an _Identity Anchor_. For each application you authenticate to using an Identity Anchor, based on that Identity Anchor, Internet Identity creates a different _pseudonym_ specific to that application. This pseudonym cannot be linked to the Identity Anchor. You can add multiple _authentication methods_, such as security keys or facial recognition on computers or phones, to an Identity Anchor, and you will be able to use each of these authentication methods to authenticate using that Identity Anchor. Also, you can create as many Identity Anchors as you wish.
 
-Unlike most authentication services, your Internet Identities do not require you to set and manage passwords or provide any personal identifying information to applications. Instead, you use the authentication profiles that you create such as facial recognition from a smartphone, your computer unlock password, or a security key.
+Unlike most authentication methods, Internet Identity does not require you to set and manage passwords or provide any personal identifying information to dapps or to Internet Identity.
 
 [[id-overview]]
 == How Internet Identity works
 
 Internet Identity takes advantage of the Web Authentication (WebAuthn) API to provide secure cryptographic authentication using smart contracts for applications running on the {IC}. 
-Internet Identity enables you to use security chips built into your smart phone or your laptop or security keys plugged into the USB port of your computer to verify your identity and authorize access to an application. 
+Internet Identity enables you to use security modules built into your smart phone, your laptop or security keys plugged into the USB port of your computer to authenticate and authorize access to dapps running on the {IC}.
 
-Applications that integrate with Internet Identity prompt you to register or log in with a previously-registered Identity Anchor. 
-Registering a device is a simple process that generates a unique cryptographic key for you that is stored on the Internet Computer blockchain. You can then use your Identity Anchor to register additional devices, so that you can access applications across all of your devices.
+Applications that integrate with Internet Identity prompt you to authenticate using an Identity Anchor. If you don't have an Identity Anchor yet, creating one is a simple process that generates a unique cryptographic key for you, whose public part is stored on the Internet Computer blockchain. You can then use your Identity Anchor to add additional devices, so that you can access applications across all of your devices.
```

Comment:
I know this hasn't changed, but the "a unique cryptographic key for you", wording sounds like "unique cryptographic key _identifiying_ you" which is a bit unfortunate.

After all the cryptographic _keys_(!!) are generated by the Yubikeys and TPM's and can be swapped out at any point. The only thing the process really generates is the number for the Identity Anchor.

Reply:
Thanks @kritzcreek . I agree we should improve this. I rewrote parts of this paragraph to:
- Make it more clear that the keys are associated to the devices that are added to an anchor (and not the anchor directly)
- Clarify that a pair of keys is generated. 
Could you take another look?